### DISCUSSION TITLE
How to input octal?

### DISCUSSION THREAD

**Discussion by q962** [freeplane/freeplane#1926] [Help]
`0777` => `777`

Why does this happen?

solution: ` 0777`

**Comment by Illegit** [comment]
Maybe you could utilize the ': --> '0777.

**Reply by q962** [reply]
`'0777` is valid.

**Comment by q962** [comment]
I see. There is an option in the settings. It is enabled by default, and it will judge the input and format it

**Comment by dpolivaev** [comment]
If Freeplane handles your input as text, as has implications on conditions because  as text 12345 is less than 9

**Reply by q962** [reply]
Oh, I guess it has something to do with filtering.

But I don't understand the point of this comparison.

Generally speaking, if I want to compare numbers, I mostly compare decimal numbers.

But what if I store hexadecimal or octal?

I want to enter octal '0777' now.

I can't imagine whether I should format it as decimal or remove the '0', it doesn't make sense.

It should be an advanced function, providing options.
For example, limit the comparison to the same base,
or format it as decimal before comparing.

I don't have this need for the time being, I don't understand the point of the comparison.

For mixed content, comparison is more difficult. More conditions need to be restricted.
I think using a script can solve it, but it becomes more difficult to use.
